libbeat/conditions/conditions.go

```diff
@@ -24,13 +24,16 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
+const logName = "conditions"
+
 // Config represents a configuration for a condition, as you would find it in the config files.
 type Config struct {
 	Equals    *Fields  `config:"equals"`
 	Contains  *Fields  `config:"contains"`
 	Regexp    *Fields  `config:"regexp"`
 	Range     *Fields  `config:"range"`
 	HasFields []string `config:"has_fields"`
+	Network   *Fields  `config:"network"`
```

Comment:
Where is this used?

Reply:
That was accidentally included. I have removed this line and squashed.